dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -0,0 +1,955 @@
+		
+			<!-- select all funding references -->
+					<xsl:variable name="funder" select="doc:metadata/doc:element[@name='project']/doc:element[@name='funder']/doc:element[@name='name']/doc:element/doc:field[@name='value']"/>
+					<xsl:variable name="funderid" select="doc:metadata/doc:element[@name='project']/doc:element[@name='funder']/doc:element[@name='identifier']/doc:element/doc:field[@name='value']"/>
+					<xsl:variable name="awardnumber" select="doc:metadata/doc:element[@name='oaire']/doc:element[@name='awardNumber']/doc:element/doc:field[@name='value']"/>
+					<xsl:variable name="awarduri" select="doc:metadata/doc:element[@name='oaire']/doc:element[@name='awardURI']/doc:element/doc:field[@name='value']"/>
+					<xsl:variable name="awardtitle" select="doc:metadata/doc:element[@name='dc']/doc:element[@name='relation']/doc:element/doc:field[@name='value']"/>
+					
+					<xsl:variable name="check_fundingreference"> 
+						<xsl:for-each select="$awardtitle">
+							<xsl:if test="$awardtitle!=''">
+								<xsl:value-of select="."/>
+							</xsl:if>
+						</xsl:for-each>
+					</xsl:variable>
+					
+					<xsl:if test="$check_fundingreference!=''">
+						<oaire:fundingReferences>
+							<xsl:for-each select="$awardtitle">
+								<xsl:if test=".!='' ">
+									<oaire:fundingReference>
+										<xsl:variable name="counter" select="position()"/>
+										<oaire:funderName>
+											<xsl:value-of select="$funder[$counter]"/>
+										</oaire:funderName>
+										<xsl:if test="$funderid[$counter]!='' ">
+											<oaire:funderIdentifier funderIdentifierType="Crossref Funder ID">
+												<xsl:value-of select="$funderid[$counter]"/>	
+											</oaire:funderIdentifier>
+										</xsl:if>
+										<xsl:if test="$awardnumber[$counter]!='' ">
+											<oaire:awardNumber>
+												<xsl:if test="$awarduri[$counter]!='' ">
+													<xsl:attribute name="awardURI">
+														<xsl:value-of select="$awarduri"/>
+													</xsl:attribute>
+												</xsl:if>
+												<xsl:value-of select="$awardnumber[$counter]"/>
+											</oaire:awardNumber>
+										</xsl:if>
+										<xsl:if test=".!='' ">
+											<oaire:awardTitle>
+												<xsl:value-of select="."/>
+											</oaire:awardTitle>
+										</xsl:if>
+									</oaire:fundingReference>
+								</xsl:if>
+							</xsl:for-each>
+						</oaire:fundingReferences>
+					</xsl:if>
+		</oaire:resource>
+	</xsl:template>
+
+	<!-- datacite:sizes -->
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_size.html -->
+	<xsl:template
+		match="doc:element[@name='bundles']/doc:element[@name='bundle']"
+		mode="datacite">
+		<xsl:if test="doc:field[@name='name' and text()='ORIGINAL']">
+			<datacite:sizes>
+				<xsl:for-each
+					select="doc:element[@name='bitstreams']/doc:element[@name='bitstream']">
+		            <xsl:value-of select="concat(doc:field[@name='size'],' bytes')"/>
```

Comment:
Should be   <datacite:size><xsl:value-of select="concat(doc:field[@name='size'],' bytes')"/></datacite:size>
Reference: https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_size.html

Reply:
Ok, noted.